### ISSUE TITLE
annotations.feather missing for some runs

### ISSUE THREAD

**Issue by ShenZheng2000** [LiljaAdam/geographical-splits#4]
In the original set, only `train/` and `val/` have `annotations.feather` for dynamic objects — `test/` doesn't. 

After geo-split, some runs without annotations end up in `val/`, and some with annotations go to `test/`, making object-level eval hard on both. 

Therefore, I wonder if this split meant only for static map elements, not for dynamic objects.

**Comment by LiljaAdam** [comment]
You are correct that samples that were originally in the test set now may be in either set (train, val, test) in the geo-splits.

If you want to train and evaluate e.g. object detection on geo-splits you would need to skip the samples originally in the test dataset. But these splits are supposed to be useful for e.g. object detection too.

I think it's good practice to tune your hyperparameters using the validation set and then evaluate the final performance on the test set. However, if you mean that there are too few val/test samples left (I have not check the numbers) I suppose you could merge the validation and test sets into one.

Hope that answers your question.
